docs/proposals/002-api-proposal/proposal.md

```diff
@@ -273,57 +255,57 @@ spec:
-- ExtProc looks up the LLMServices affiliated with this pool `examplePool`
+- ExtProc looks up the InferenceModels affiliated with this pool `examplePool`
 - `name-generator` is currently undergoing a change of LoRA adapters from `name-generator-v3` (20% traffic split) to `name-generator-v2` (80% traffic split)
 - `name-generator-v2` is selected as the LoRA adapter, and replaces `name-generator` in the body of the request (mutated by ext-proc) 
 - the request is then efficiently scheduled onto one of the valid Pods
 - Prometheus metrics are sent back to the LSP, aggregated and re-emitted via sidecar (following the metric standardization)
 
-How Multiple LLMServerPools might integrate together:
+How Multiple InferencePools might integrate together:
 
-![K8s Gateway with LLMServerPools](./images/gw_w_lsp.svg)
+![K8s Gateway with InferencePools](./images/gw_w_lsp.svg)
 
 Here we see that we can have:
 - Multiple Routes pointing to the same pool
 - Routes splitting traffic across multiple pools
 
-The functionality of the Kubernetes Gateway is unchanged with this proposal, allowing seamless integration with the LLMServerPool.
+The functionality of the Kubernetes Gateway is unchanged with this proposal, allowing seamless integration with the InferencePool.
 
 
 ### Alternatives
 
 #### Key Decisions
 
 Our alternatives hinge on some key decisions:
-- Allowing HTTPRoute to treat the LLMServerPool as the backendRef
-  - Whereas the alternatives might have the LLMService as the backend ref
+- Allowing HTTPRoute to treat the InferencePool as the backendRef
+  - Whereas the alternatives might have the InferenceModel as the backend ref
 - Creating a separate layer of abstraction, instead of extending HTTPRoute
   - Explained in more detail in the LLMRoute section
 
-#### LLMService as a backend ref
+#### InferenceModel as a backend ref
 
-We toyed with the idea of allowing an LLMService be the target of an HTTPRouteRules backend ref. However, doing so would require the Kubernetes Gateway to be able to interpret body level parameters (assuming OpenAI protocol continues to require the model param in the body), and require that the HTTPRoute also specify the backend the LLMService is intended to run on. Since we our primary proposal already specifies the backend, packing this functionality would require substantial work on the Kubernetes Gateway, while not providing much flexibility.
+We toyed with the idea of allowing an InferenceModel be the target of an HTTPRouteRules backend ref. However, doing so would require the Kubernetes Gateway to be able to interpret body level parameters (assuming OpenAI protocol continues to require the model param in the body), and require that the HTTPRoute also specify the backend the InferenceModel is intended to run on. Since we our primary proposal already specifies the backend, packing this functionality would require substantial work on the Kubernetes Gateway, while not providing much flexibility.
```

Comment:
```suggestion
We toyed with the idea of allowing an InferenceModel be the target of an HTTPRouteRules backend ref. However, doing so would require the Kubernetes Gateway to be able to interpret body level parameters (assuming OpenAI protocol continues to require the model param in the body), and require that the HTTPRoute also specify the backend the InferenceModel is intended to run on. Since our primary proposal already specifies the backend, packing this functionality would require substantial work on the Kubernetes Gateway, while not providing much flexibility.
```